Eos/Soave-Redlich-Kwong/EOS.H

```diff
@@ -414,13 +414,14 @@ REY2T(amrex::Real R, amrex::Real E, amrex::Real Y[], amrex::Real& T)
   EOS::MixingRuleBm(Y, bm);
   K1 = (1.0 / bm) * log(1.0 + bm*R); 
 
-  // Use input T as initial guess, but constrain it to reasonable bounds
-  Tn = amrex::min(amrex::max(T,300.0),5000.0);
+  // Use input T as initial guess
+  Tn = T;
+  // constrain it to reasonable bounds
+  // Tn = amrex::min(amrex::max(T,300.0),5000.0);
```

Comment:
No more protection here?

Reply:
I had second thoughts about hard coding what reasonable bounds are - for example Michael Martin ran a supercritical helium case at ~20 K or something. I think a happy medium here is asserting that the temperature be positive and other than that putting the onus on the user to supply a reasonable guess. I made the corresponding changes in the latest commit.

Reply:
The other approach is to ParmParse these in (with sensible defaults) at the beginning of the run.  again, no emergency, but if doing so allows a completely different app to use without hacking code, it seems like a small price to pay.